src/Components/Feelings/Feelings.js

```diff
@@ -1,40 +1,23 @@
 import React from 'react';
 import './Feelings.css';
+import feelingsData from './feelingsData';
 import { Link } from 'react-router-dom';
 
-const Feelings = ({ setFeeling }) => {
+const Feelings = () => {
 
-  const selectFeeling = ({ target }) => {
-    setFeeling(target.classList[1]);
-  }
+  const feelingsLinks = feelingsData.map(feelingData => {
+    return feelingData.associatedFeelings.map(feel => {
+      return <Link to={`/what-should-you-do/${feel}`} key={feel} className='feeling'>{feel}</Link>
```

Comment:
YES so beautiful

Reply:
Amber, this is such an elegant way to solve the loose clicking problem. I'm really impressed! Thank you

Reply:
HUGE! Thank you Amber!